metals/src/main/scala/scala/meta/internal/metals/ScalaVersions.scala

```diff
@@ -24,7 +24,7 @@ object ScalaVersions {
   val isLatestScalaVersion: Set[String] =
     Set(BuildInfo.scala213, BuildInfo.scala212, BuildInfo.scala211)
 
-  def recommendedVersion(scalaVersion: String): String = BuildInfo.scala212
+  def recommendedVersion: String = BuildInfo.scala212
```

Comment:
The change here should most likely be to use the `scalaVersion` to get the binary version and the recommended version for this binary version. It would be BuildInfo.scala212 for 2.11 and 2.12. BuildInfo.scala213 for 2.13.

Reply:
Just reverted the initial change, and added a commit to do that.